index.html

```diff
@@ -1,63 +1,205 @@
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Multi-Post Stories
+        </h1>
+        <ul class="client-role-year">
+          <li class="work-client">CANOPY</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-role">Back End Dev</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
+        </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
       </div>
-    </section>
-  </body>
+    </div>
+  </section>
+  <section id="about-myself">
+    <div id="about-myself-container">
+      <h1 class="title" id="headline-title">
+        About Myself
+      </h1>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h2 class="supporting-text">LET’S CONNECT</h2>
+      <ul id="social-media-container">
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Twitter's icon" src="./visuals/twitter-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="LinkedIn's icon" src="./visuals/linkedin-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Medium's icon" src="./visuals/medium-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="GitHub's icon" src="./visuals/github-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="AngelList's icon" src="./visuals/angellist-icon.png" />
+          </a>
+        </l1>
+      </ul>
+      <a class="button" href="#">Get my resume</a>
+    </div>
+    <ul id="boxes">
+      <li class="box" id="box-languages">Languages<img class="chevron-icon" src="./visuals/chevron-down-icon.png"/></li>
+      <ul class="box-list" id="languages">
+        <li class="box-element"><img class="language-icon" src="./visuals/javascript-icon.png"/>JavaScript</li>
+        <li class="box-element"><img class="language-icon" src="./visuals/html-icon.png"/>HTML</li>
+        <li class="box-element"><img class="language-icon" src="./visuals/css-icon.png"/>CSS</li>
+      </ul>
+      <li class="box">Frameworks<img class="chevron-icon" src="./visuals/chevron-right-icon.png"/></li>
+      <li class="box">Skills<img class="chevron-icon" src="./visuals/chevron-right-icon.png"/></li>
+    </ul>
```

Comment:
Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with a detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
      </li>
      <li>
        Skills
      </li>
    </ul>
```

## CONCLUSION:-

Kindly make the list of languages inside the li tag as presented in the above code.